### DISCUSSION TITLE
Q: I've lost my phone and I need to get the setup code again!

### DISCUSSION THREAD

**Discussion by anasouardini** [community/community#141616] [Code Security]
### Select Topic Area

Question

### Body

I've used my recovery codes (two of them) and I realized I can't use them forever. Please, help!

**Comment by ghostinhershell** [comment]
Hi @anasouardini! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
